openlibrary/templates/type/user/view.html

```diff
@@ -1,9 +1,12 @@
 $def with (page)
 
 $var title: $page.displayname
+$ username = page.key.split('/')[-1]
```

Comment:
Is there a safer way to get this, such as from `get_users_settings`? Or is this way of doing it considered pretty safe?

Reply:
I think this should be okay in this case.